### DISCUSSION TITLE
Do we have to reclone the repo again after cleaning ?

### DISCUSSION THREAD

**Discussion by sebastienroux** [newren/git-filter-repo#188] [Q&A] [answered]
Hi,

We are 10 devs on the same projects, if i use git-flter-repo to delete some files and modify somes emails, other developers have to reclone again, or just pull the changes ? 

Thx

**Comment by newren** [comment, marked as answer]
You can get away without recloning, but it is very likely to lead to mistakes and people merging both the old bad history with the new good history and pushing it up, making everything a mess.  I do not recommend it; it's a much better idea to have everyone reclone.

See items 4 & 5 under the "Discussion" section of the manual (https://htmlpreview.github.io/?https://github.com/newren/git-filter-repo/blob/docs/html/git-filter-repo.html#DISCUSSION) for details.
